### PR TITLE
Iar 22/tomas/create a posts feed db

### PR DESCRIPTION
**Pull request by TomasBenes**

### CODE REVIEW

File: src/services/PostsService.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@Service()
+export class PostsService {
+
+	constructor(
+		private postsRepository: PostsRepository,
+	) {}
+
+	public async create(posts: Posts): Promise<void> {
+		await this.postsRepository.save(posts);
+	}
```
Comment (Karhalll) [inline src/services/PostsService.ts:15]: Not needed for this ticket but looks great :)

File: src/services/SubredditsService.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	) {}
+
+	public async create(subreddits: Subreddits): Promise<void> {
+		await this.subredditsRepository.save(subreddits);
+	} 
+	
+	public async findAll(): Promise <Subreddits[] | undefined> {
+		return await this.findAll();
+	}
+}
```
Comment (Karhalll) [inline src/services/SubredditsService.ts:20]: Not needed for this ticket but looks great :)

File: src/services/UserService.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export class UserService {
 		private encryptService: EncryptService
 	) {}
 
-  public async create(userCreation: UserCreation): Promise<void> {
+	public async create(userCreation: UserCreation): Promise<void> {
```
Comment (Karhalll) [inline src/services/UserService.ts:19]: Thx for correction of my indentation. :P

**Review by Karhalll — APPROVED** [review]
Looking great and simple. Some "future" code unneeded right now but ok.

**Review by SimProch — APPROVED** [review]

**Pull request merged**